ironfish/src/sdk.ts

```diff
@@ -242,6 +242,10 @@ export class IronfishSdk {
       }
     }
 
+    if (this.config.get('enableAssetVerification')) {
+      node.assetsVerifier.start({ enableRefresh: false })
+    }
```

Comment:
It looks like this should still happen even if the person isn't using the SDK. I would recommend starting all sub-systems in the node rather than here.

Reply:
This system is already started when you call the `IronfishNode.start()` method, and it's started with `enableRefresh: true` (the default).

I interpreted this method here (`IronfishSdk.node()`) to be a "light" version of the node, therefore I put here the lines of code to start a lightweight assets verifier. Is my understanding wrong? In order to put this code in the `IronfishNode`, I would need to put it in the `init()` method. This would be the only service that is started at `init()`. To make things more clean, perhaps I can add a `IronfishNode.lightweightStart()` (or similar) and call it here. Thoughts about that?